src/components/card-section/featured-card-section.tsx

```diff
@@ -53,7 +53,10 @@ const FeaturedCardSection: React.FunctionComponent<
                     {...getCardProps(content[1], 'small')}
                 />
                 <Card
-                    sx={smallFeaturedCardStyles}
+                    sx={{
+                        ...smallFeaturedCardStyles,
+                        display: ['none', null, null, 'block'],
+                    }}
```

Comment:
Looks like it's working great on the Content Type pages! But I think this behavior should only take effect when `featuredCardType` is `medium`. When it is set to `large`, the alignment gets thrown off on tablet size. Here's an example of that on the JavaScript page.
<img width="353" alt="Screen Shot 2023-01-13 at 4 21 26 PM" src="https://user-images.githubusercontent.com/46664803/212421905-9e2c9001-b59f-42d9-980e-1caec9c1cd90.png">

I think you could just fix it by only using the `'none'` on that first breakpoint when it's `medium` and `'block'` when it's `large`.